Exercise_4.py

```diff
@@ -8,13 +9,32 @@ def __init__(self, data):
           
 """ Inorder traversal of a binary tree"""
 def inorder(temp): 
-  
-    
+  if temp:
+      inorder(temp.left)
+      print(temp.key)
+      inorder(temp.right)    
   
   
 """function to insert element in binary tree """
-def insert(temp,key): 
-  
+def insert(temp,key):
+    #print(temp) 
+    list = []  
+    list.append(temp)   
```

Comment:
Please include edge case for insertion of root node, when the tree doesn't exist.

Reply:
Hi Samarth,

Thanks for reviewing the exercises.
I have updated exercise 4 to include the edge case but I am getting one issue, on printing the inorder traversal:
Result are:(Case where only key =12 is getting inserted)
Inorder traversal before insertion: None
Inorder traversal after insertion: 12  None
As per my understanding None should not be present. Please let me know where I have made the mistake

Thanks in advance.

-Garima